R/lint.R

```diff
@@ -640,16 +640,16 @@ sarif_output <- function(lints, filename = "lintr_results.sarif") {
     as.character(utils::packageVersion("lintr"))
   sarif$runs[[1L]]$originalUriBaseIds$ROOTPATH$uri <- ""
   rule_index_exists <- FALSE
-  root_path_uri <- gsub("\\\\", "/", package_path)
+  root_path_uri <- gsub("\\\\", "/", package_path, fixed = TRUE)
```

Comment:
please double-check it's doing the same thing, I think the fixed equivalent should be just `"\\"`